sdk/hybridcompute/Microsoft.Azure.Management.HybridCompute/src/Microsoft.Azure.Management.HybridCompute.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HybridCompute</PackageId>
+    <Description>Provides developers with libraries for managing hybrid machines</Description>
+    <Version>0.0.1</Version>
+    <AssemblyName>Microsoft.Azure.Management.HybridCompute</AssemblyName>
+    <PackageTags>management;hybridcompute;</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+        This is private preview release of the Azure Hybrid Compute SDK.
```

Comment:
Hey @farehar , are you sure if this is for private preview? It should be against azure/sdk-for-net-pr repo if so.

Reply:
@isra-fel: You are right, this is for our public preview.